src/types/UiaTextRangeBase.cpp

```diff
@@ -953,7 +953,7 @@ void UiaTextRangeBase::_moveEndpointByUnitCharacter(_In_ const int moveCount,
             }
             break;
         case MovementDirection::Backward:
-            success = buffer.MoveToPreviousGlyph(target, allowBottomExclusive);
+            success = buffer.MoveToPreviousGlyph(target);

```

Comment:
this is because it is safe to move backwards off the exclusive end, and you're checking before loading the glyph data that it _is_ a valid position?

Reply:
Correct. It's safe to move backwards off the exclusive end. So let's not bother passing in `allowBottomExclusive` and _always_ allow it.